test/pkg/test/framework.go

```diff
@@ -200,37 +189,11 @@ func (env *Framework) ExpectToBeClean(ctx context.Context) {
 					//so we temporarily skip to verify whether ServiceExport created TargetGroup is deleted or not
 					continue
 				}
-				g.Expect(env.TestCasesCreatedServiceNames).To(Not(ContainElements(BeKeyOf(*tg.Name))))
 			}
 		}
 	}).Should(Succeed())
 }
 
-// if we still want this method, we should tag the things we create with the test suite
-func (env *Framework) CleanTestEnvironment(ctx context.Context) {
```

Comment:
IMO the "best" way to clean up the test environment is via the new e2e-clean target. Tracking these resources as side-effects of various creation calls seems unreliable, similar to the reasons we're eliminating the ```LatticeDataStore``` cache.

Given we're not using the cleanup methods today, I think removing them is better for now than continuing to maintain this unused code.

Reply:
Interesting idea, do you want to remove cleanup after each test and do only one at the very end of all tests?

Reply:
Removing resources as each test finishes is what it does today. We don't call anything at the end of all tests. These ```Clean*``` methods are currently unused.